packages/jsii-pacmak/test/targets/version-utils.test.ts

```diff
@@ -63,6 +63,18 @@ const examples: Record<
     nuget: '[,0.1.2]',
     python: '<=0.1.2',
   },
+
+  // Somewhat unusual ranges
+  '*': {
+    maven: '(0.0.0,]',
+    nuget: '(0.0.0,]',
+    python: '>=0.0.0',
+  },
+  '^1.2.*': {
```

Comment:
Is this a valid `semver` range notation? It appears confusing to me as two different intentions are expressed:
- `1.2.*` is typically used to mean minor-pinned (`>= 1.2.0, <= 1.3.0-0`)
- `^1.2.0` means major-pinned (`>= 1.2.0, < 2.0.0-0`)

Reply:
Great callout! Switched it to `1.2.*` :)